src/modules/core/components/Popover/Popover.tsx

```diff
@@ -253,10 +262,14 @@ const Popover = ({
   // Timeouts
   useEffect(() => {
     const currentOpenTimeoutRef = openTimeoutRef && openTimeoutRef.current;
+    const currentCloseTimeoutRef = closeTimeoutRef && closeTimeoutRef.current;
     return () => {
       if (window && currentOpenTimeoutRef) {
         window.clearTimeout(currentOpenTimeoutRef);
       }
+      if (window && currentCloseTimeoutRef) {
+        window.clearTimeout(currentCloseTimeoutRef);
+      }
```

Comment:
Again, I see no reason to use `window.clearTimeout` as opposed to just `clearTimeout`